src/component/footer/footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./footer.module.scss";
+
+const Footer = ({
+  handleClickBack,
+  handleClick,
+  previousPageLink,
+  nextPageLink,
+}) => {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.footer__backoptions} onClick={handleClickBack}>
+        <div className={styles.footer__arrowleft} />
+        {previousPageLink}
+      </div>
+      <div className={styles.footer__previousoptions} onClick={handleClick}>
```

Comment:
If there are two props, please code like this:
<div
  className={styles.footer__previousoptions}
  onClick={handleClick}
>